service-admin/src/App/templates/error/no-user-details-error.phtml

```diff
@@ -0,0 +1,9 @@
+<?php $this->layout('layout::default', ['title' => 'Incomplete account details']) ?>
+
+<h1>Oops!</h1>
+<h2>This is awkward.</h2>
+<p>You need to complete your LPA account details first!</p>
+<p>
+    You must fill in your account details on the LPA online service before accessing the Admin service.
+</p>
+<p> Click <a href="https://www.lastingpowerofattorney.service.gov.uk/login">here</a> to sign into your LPA account and complete your details.</p>
```

Comment:
A better way to do linking in HTML documents is to avoid the use of clicking "here" and instead explain what the link does. For instance;

"You will need to [sign in using your account] on the LPA frontend and complete your details".

Sadly we don't have a content person anymore to take a look at this but theres lots on the web [that you might want to read](http://www.leafdigital.com/class/lessons/writing/9.html).